nilearn/plotting/html_surface.py

```diff
@@ -195,12 +204,16 @@ def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
         volume_img=stat_map_img, mesh=surf_mesh, threshold=threshold,
         cmap=cmap, black_bg=black_bg, vmax=vmax, vmin=vmin,
         symmetric_cmap=symmetric_cmap)
+    info['colorbar'] = colorbar
+    info['cbar_height'] = cbar_height
+    info['cbar_fontsize'] = cbar_fontsize
```

Comment:
Same here, make it `colorbar_height` and so on.

Reply:
There are are more instances of cbar_..., this comment applies to all of them.

Reply:
I did it for all python names. the dictionary keys are for javascript functions in the generated pages, never seen by users so I didn't change them to avoid changing the javascript